packages/wallet/src.ts/index.ts

```diff
@@ -175,12 +221,65 @@ export class Wallet extends Signer implements ExternallyOwnedAccount, TypedDataS
 
 	signTransaction(transaction: TransactionRequest): Promise<string> {
 		this._checkAddress('signTransaction');
-		this.checkTransaction(transaction);
-		return this.populateTransaction(transaction).then(async readyTx => {
-			const tx = serializeHederaTransaction(readyTx);
-			const pkey = HederaPrivKey.fromStringECDSA(this._signingKey().privateKey);
+		if (transaction.from) {
```

Comment:
This was alredy implemented and merged in the `develop` branch. It seems that you are reverting the change with this PR